scss/_variables.scss

```diff
@@ -1345,7 +1346,7 @@ $dropdown-spacer:                   $border-width * -1 !default;
 $dropdown-font-size:                $font-size-base !default;
 $dropdown-color:                    var(--#{$prefix}body-color) !default;
 $dropdown-bg:                       $white !default;
-$dropdown-border-color:             $gray-500 !default;
+$dropdown-border-color:             var(--#{$prefix}border-color-translucent) !default;
```

Comment:
I think that many other border-color could be updated this way in Boosted at least

Reply:
Yes not done in this PR to avoid too much changes to review. Could be tackled in a PR just after.